containers/ecr-viewer/src/app/signin/signin.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import { Button } from "@trussworks/react-uswds";
```

Comment:
for consistency with other routes, could we put this in a folder called `components` and name it `Redirect`. Or maybe just make the button a component and move the rest of this to the page? It doesn't seem reusable in general and the button is the only thing that needs to be client side as far as I can tell